homeassistant/components/sonos/__init__.py

```diff
@@ -208,6 +209,14 @@ def is_device_invisible(self, ip_address: str) -> bool:
 
     async def async_subscribe_to_zone_updates(self, ip_address: str) -> None:
         """Test subscriptions and create SonosSpeakers based on results."""
```

Comment:
There a multiple ways to get into this function including the autodiscovery path which is being blocked below. I want this in here in case we start getting Sonos topology messages with V6 addresses.